ambari-logsearch/ambari-logsearch-server/src/main/java/org/apache/ambari/logsearch/model/request/LogParamDefinition.java

```diff
@@ -30,15 +30,15 @@
 
 public interface LogParamDefinition {
```

Comment:
What is the purpose of this interface? I found implementations only which are data like classes. I have not found any classes/methods which are depend from the interface itself.